client/src/App.css

```diff
@@ -23,6 +23,48 @@
   color: #61dafb;
 }
 
+.text-align-center {
+  text-align: center;
+  top: 50%;
+  line-height: 95px !important;
+}
+.left-margin-25 {
+  margin: 10px 0 0 25px;
+}
+
+.text-color-white {
+  color: white;
+}
+
+.board {
```

Comment:
this files should contain only common CSS.
Put component-styles in component's css file.